src/app/components/room/room.service.js

```diff
@@ -353,10 +428,27 @@
       var display = FeedsService.findMain().display;
       var connection;
       var id;
-
+      
+      // emit `screenshare` event 
+      jhEventsProvider.emitEvent({
+        type: "screenshare",
```

Comment:
This event also looks very influenced by callstats API. I don't think we need a different event for publishing screensharing and for publishing camera, just a different argument to the same type of event. From the jangouts point of view both are basically the same event even if callstats has a dedicated screenshare start/stop events.